src/bin/find.js

```diff
@@ -32,9 +33,12 @@ Object.keys(options).forEach(function findRules(option) {
   var ruleFinderMethod = ruleFinder[option]
   if (argv[option] && ruleFinderMethod) {
     rules = ruleFinderMethod()
+    argv.verbose && console.log( // eslint-disable-line no-console
```

Comment:
Something to be considered as a follow up.
[cyclomatic complexity](https://en.wikipedia.org/wiki/Cyclomatic_complexity) of `findRules` function has increased over the time, it should be refactored. 
**Not in this PR**